opencti-platform/opencti-graphql/src/database/data-initialization.js

```diff
@@ -62,6 +62,7 @@ export const SETTINGS_CAPABILITIES = {
     { name: 'SETACCESSES', description: 'Manage credentials', attribute_order: 3200 },
     { name: 'SETMARKINGS', description: 'Manage marking definitions', attribute_order: 3300 },
     { name: 'SETLABELS', description: 'Manage labels & Attributes', attribute_order: 3400 },
+    { name: 'SECURITYACTIVITY', description: 'Security Activity', attribute_order: 3500 },
```

Comment:
The description "Security Activity" doesn't look very clear, we usually name the capabilities with a verb like "access" or "manage". "Access security activity" ? to validate with product.

Reply:
I like your idea, I wasn't sure of what to put here in the description. I like Access Security Activity, will ask product about it